bundles/pom.xml

```diff
@@ -71,6 +67,7 @@
     <module>org.openhab.binding.automower</module>
     <module>org.openhab.binding.avmfritz</module>
     <module>org.openhab.binding.awattar</module>
+    <module>org.openhab.binding.bambulab</module>
```

Comment:
This is the only change that should be made to this file. All other changes (remove empty lines etc) should be reverted.